screens/Browse.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import {
+    StyleSheet,
+    Text,
+    View,
+    SafeAreaView,
+    ScrollView,
+    Button,
+} from 'react-native';
+import { PageHeader } from '../components/PageHeader';
+import {meals} from "../components/CategoryData"
+export default function Browse({ navigation, route }) {
+    const [term, setTerm] = useState("");
+    const [searchApi, results, errorMessage] = useResults();
+    const filterResultsByPrice = price => {
+        // price === '$' || '$$' || '$$$'
+        console.log(results)
+        return results.filter(result => {
+          return result.price === price;
+        });
+      };
+    return (
+        <SafeAreaView style={{ backgroundColor: '#eee', flex: 1 }}>
+            <PageHeader title="Browse" navigation={navigation} />
+            <View style={{marginTop:2, marginBottom:1, marginRight:4, position:'relative', justifyContent:'center'}}>
+                    <Ionicons name="search-sharp" size={20} color="black" style={{zIndex:10, position:'absolute',alignSelf:'center',left:16 }} />
+                    <TextInput style={styles.input,{borderRadius:10,backgroundColor:'#C0C0C0',paddingBottom:8,paddingTop:8, paddingLeft:20,paddingLeft:36, paddingRight:20}} value={term} onChangeText={setTerm} onEndEditing={()=>searchApi(term)}placeholder="Search anything" />
```

Comment:
Should we use StyleSheet instead of inline styles? I think it would look cleaner